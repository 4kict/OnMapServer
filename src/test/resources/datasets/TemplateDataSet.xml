<?xml version="1.0" encoding="UTF-8"?>

<dataset>
    <User id="-111" city="Moskaw" country="russia" hashkey="123456789" name="Vasia"/>
    <Mesage id="-222" autorId="-112" localRowId="77" mesaga="Just Some messAge" msgType="1" recip_id="-111"
            objVersion="1"/>

    <User id="-112" city="Numbai" country="nigeria" hashkey="-1031302084" name="Lumumbu" objVersion="1"
          lastGeoDecode="1"/>

</dataset>