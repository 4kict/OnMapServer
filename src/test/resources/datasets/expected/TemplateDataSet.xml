<?xml version="1.0" encoding="UTF-8"?>

<dataset>
    <User id="-111" city="Moskaw" country="russia" hashkey="123456789" name="Vasia"/>
    <Mesage id="-222" autorId="-112" localRowId="77" mesaga="Just Some messAge" msgType="1" recip_id="-111"
            objVersion="1"/>


    <User id="-112" city="Numbai" country="nigeria" hashkey="9876543321" name="Lumumbu" objVersion="1"/>
    <Mesage autorId="-111" mesaga="someMessage from Vasia to Mumbai" recip_id="-112"/>

</dataset>